src/core/thread/key_manager.cpp

```diff
@@ -48,25 +49,6 @@ const uint8_t KeyManager::kThreadString[] = {
     'T', 'h', 'r', 'e', 'a', 'd',
 };
 
-const otMasterKey KeyManager::kDefaultMasterKey = {{
-    0x00,
-    0x11,
-    0x22,
-    0x33,
-    0x44,
-    0x55,
-    0x66,
-    0x77,
-    0x88,
-    0x99,
-    0xaa,
-    0xbb,
-    0xcc,
-    0xdd,
-    0xee,
-    0xff,
-}};
```

Comment:
Some of the tests may rely on the default values and/or verify them.

For example, I know, `test-002-form` under troanj does verify the default values  and checks once we use `wpantund` `form` command, the default values are not used by newly formed Thread network (i.e. new random values are used for key, pan-id, xpanid, (note that this is done by `wpantund` during `form`)).  

 With the change from this PR, we probably can remove some of the checks  in the test-002.

Reply:
Let's make @qilinkang aware of this change too.

Reply:
It seems the default key verification in `test-002-form` is the only thing relying on the default value in toranj tests. Removed it and toranj tests passed.